scripts/source_files/release_functions.sh

```diff
@@ -1,17 +1,28 @@
 #!/usr/bin/env bash
 
-function find_gradle_property(){
-    [[ "$(declare -p $1 2>/dev/null)" =~ "declare -a" ]] || return 11
-    local -n output_target=$1
-    for command_line_argument in "$@"; do
-        if [[ "${command_line_argument}" =~ ^-P.*=.* ]];then
-          output_target+=("${command_line_argument}")
-        fi
-    done
+function find_gradle_property() {
+  [[ "$(declare -p $1 2>/dev/null)" =~ "declare -a" ]] || return 11
+  local -n output_target=$1
+  for command_line_argument in "$@"; do
+    if [[ "${command_line_argument}" =~ ^-P.*=.* ]]; then
+      output_target+=("${command_line_argument}")
+    fi
+  done
 }
 
-function releaseRepositoryAndPushVersion()
-{
+function push_version() {
+  echo Pushing version and tag to GitHub repository...
+  git push && push_result="success" || push_result="failure"
+  if [ push_result != "success" ]; then
+    branch_name="release/${VERSION_PREFIXED}"
```

Comment:
Maybe it's too early in the morning, but where is the VERSION_PREFIXED defined? 
Further, at the current point the clean-up of this branch has to be done manually, doesn't it?

Reply:
Ah, and this branch also has to be provided to other scripts somehow, I guess?

Reply:
VERSION_PREFIXED is defined in the release.sh. Not the best design, I know...

Reply:
> Ah, and this branch also has to be provided to other scripts somehow, I guess?

Good catch. I have not thought of that!